pkg/component/etcd/bootstrap.go

```diff
@@ -334,11 +336,16 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 			serviceAccount,
 			clusterRole,
 			clusterRoleBinding,
-			podDisruptionBudget,
 			vpa,
 		}
 	)
 
+	if versionutils.ConstraintK8sGreaterEqual126.Check(b.kubernetesVersion) {
+		podDisruptionBudget.Spec.UnhealthyPodEvictionPolicy = &unhealthyPodEvictionPolicyAlwatysAllow
+	}
+
+	resourcesToAdd = append(resourcesToAdd, podDisruptionBudget)
```

Comment:
Any reason why `podDisruptionBudget` is now handled specially, i.e. added separately to `resourcesToAdd`? After the `var` block only one attribute is changed, but not the object reference itself.

Reply:
Just saw that it was done this way for service and deployment. I could change it if you want.

Reply:
I was just surprised by the somewhat unnecessary change, but if this is how it is handled in this file. Let's stay in sync with the remaining code.